dispatch/static/manager/src/js/pages/EventAudit/EventCard.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+
+import { AnchorButton, Tooltip } from '@blueprintjs/core'
+import { LinkButton } from '../../components/inputs'
+import { humanizeDatetime } from '../../util/helpers'
+
+function labelAndProp(label, prop) {
```

Comment:
Convert this to a functional React component, then call like so below:

`<EventLabel label='Description' value={this.props.event.description} />`